### DISCUSSION TITLE
Glossary - just use See Also section?

### DISCUSSION THREAD

**Discussion by hamishwillee** [mdn/mdn-community#569] [Content]
Glossary page template has an additional info section that looks like this:

![image](https://user-images.githubusercontent.com/5368500/133720255-93c05f9b-3d78-4bd1-9b0b-1a7e858b06bb.png)

Seems like an unhelpful and inconsistent distinction. As I'm in there anyway removing the "Quicklinks" sections , any objection to me just making the standard section for all Glossary items be "See also"?

Nothing against people providing more refined sub headings, but I think they are unlikely to be necessary in the majority of cases.

@wbamberg @teoli2003

**Comment by wbamberg** [comment]
I'm in favour of this, for sure! I think I might have had the same thought recently but didn't manage to hold onto it :).

**Reply by hamishwillee** [reply]
Great! Just FYI, the current plan is that the Glossary sidebar will be auto-generated without any input from the page (probably as the entire glossary, nested under heading A, B, C ..

**Reply by hamishwillee** [reply]
FYI PR here: https://github.com/mdn/content/pull/9079

**Comment by pepelsbey** [comment]
Closing as answered/settled.
